projects/python/perception/pose_estimation/high_resolution_pose_estimation/demos/benchmarking_demo.py

```diff
@@ -28,11 +28,12 @@
                         action="store_true")
     parser.add_argument("--height1", help="Base height of resizing in heatmap generation", default=360)
     parser.add_argument("--height2", help="Base height of resizing in second inference", default=540)
-
+    parser.add_argument("--method", help="Choose between primary or adaptive ROI selection methodology defaults to adaptive",
+                        default="adaptive")
```

Comment:
This constrains the selection between the only valid ones, which is helpful when --help is used and avoids errors down the line when a wrong parameter is passed.

```suggestion
                        default="adaptive", choices=["primary", "adaptive"])
```